tests_integration/test_data/projects/napp_init_v1/app/README.md

```diff
@@ -0,0 +1,4 @@
+# README
+
+This directory contains an extremely simple application that is used for
+integration testing SnowCLI.
```

Comment:
You may want to document that this and the corresponding _v2 are meant to be identical, save for PDF version, and that they're just the result of running `snow app init --template basic` (basic is default)